src/ui/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@
 qt_add_qml_module(mozillavpn-ui
     VERSION 1.0
     URI Mozilla.VPN
+    NO_PLUGIN_OPTIONAL
```

Comment:
Ok, so here is the thing: this fixes the issue and I don't know why.

With the removal of qmldir files manually written by developers
and the usage of qt_add_qml_module, it's not like there will not
be any more qmldir files, it's that they will be generated for us.

When I looked at the generated files I noted they have this in the header

```
linktarget mozillavpn-uiplugin
plugin mozillavpn-uiplugin
classname Mozilla_VPNPlugin
typeinfo mozillavpn-ui.qmltypes
prefer :/Mozilla/VPN/
```

These are not defined in the lottie and nebula qmldir files -- which work.

I started playing around with options that would change this such as
NO_PLUGIN (which did nothing) and NO_PLUGIN_OPTINAL which fixed the import
issues we were having :shrug.

Reply:
Oh yeah, just for the record. After I aded this property
this is what the header looks like:

```
linktarget mozillavpn-uiplugin
plugin mozillavpn-uiplugin
classname Mozilla_VPNPlugin
typeinfo mozillavpn-ui.qmltypes
prefer :/Mozilla/VPN/
```